src/test/java/no/unit/nva/cristin/projects/FetchCristinProjectsTest.java

```diff
@@ -243,14 +251,27 @@ void getsCorrectUriWhenCallingQueryProjectsUriBuilder() throws Exception {
             NUMBER_OF_RESULTS, DEFAULT_NUMBER_OF_RESULTS);
         URI uri = new URI(QUERY_CRISTIN_PROJECTS_EXAMPLE_URI);
 
-        assertEquals(uri, cristinApiClientStub.generateQueryProjectsUrl(params));
+        assertEquals(uri, cristinApiClientStub.generateQueryProjectsUrl(params, QUERY_USING_TITLE));
+    }
+
+    @Test
+    void getsCorrectUriWhenCallingQueryGrantIdUriBuilder() throws Exception {
```

Comment:
if this is a test for the stub it should be clarified in the test (i.e.. "stubGets..."). Maybe it would be better to be moved in own class